letsencrypt/client/network2.py

```diff
@@ -90,6 +88,9 @@ def _check_response(cls, response, content_type=None):
                         response_ct)
 
                 try:
+                    # TODO: This is insufficient or doesn't work as intended.
```

Comment:
As you noted in another comment... this is a problem with Boulder.  I wrote the comment just when I realized that the error details weren't being included. 

Right now I just decided to print out everything... :/